RetailCoder.VBE/Common/ModuleExporter.cs

```diff
@@ -10,17 +10,11 @@ public class ModuleExporter : IModuleExporter
     {
         public bool TempFile { get; private set; }
 
-        public string ExportPath
-        {
-            get
-            {
-                return TempFile
-                    ? ApplicationConstants.RUBBERDUCK_TEMP_PATH
-                    : Path.GetDirectoryName(Assembly.GetAssembly(typeof(App)).Location);
-            }
-        }
+        public string ExportPath => TempFile
+            ? ApplicationConstants.RUBBERDUCK_TEMP_PATH
+            : Path.GetDirectoryName(Assembly.GetAssembly(typeof(App)).Location);
 
-        public string Export(IVBComponent component, bool tempFile = false)
+        public string Export(IVBComponent component, bool tempFile = true)
```

Comment:
this is a pretty breaking change... are we sure this doesn't affect any callers incorrectly?